### DISCUSSION TITLE
ACG homepage

### DISCUSSION THREAD

**Discussion by sahil5d** [a-cosmology-group/acg#224] [A Cosmology Group]
Hi everyone,

Please review the latest edits to ACG's homepage letter of introduction.

https://github.com/a-cosmology-group/acg/pull/219
https://github.com/a-cosmology-group/acg/pull/219/files

https://cosmology.info

**Comment by RedshiftDrift** [comment]
Thanks for the edits @sahil5d, I did leave many typos in the original text.

The changes are good overall, but I have two concerns:
- Although relativity suffers from interpretation problems and is certainly not valid in the high-gravity regimes (black holes, neutron stars, the universe, etc...), the theory has given correct predictions ***without any adjustments to the theory*** for every laboratory experiments made during the 100 years since Einstein came up with it.  Many ACG members don't seem to appreciate the difference between 'interpretation' and 'predictions'.  (The same is true for quantum mechanics.)
On the other hand, plasma physics has evolved considerably in the last 100 years since it is a more of a 'phenomenological' theory.  For example, Hans Alfvén has discovered many new effects in plasmas.  Plasma physics is good physics, but it is by no means a stable fundamental theory.
My concern with your edit is that a group called "the Thunderbolts Project" has used the excellent work done in plasma physics to promote non-scientific ideas on the nature of the electric universe.  ACG should avoid at all costs any association with the "Thunderbolt Project - Electric Universe" charlatans.  'Relativity' was a better example of a stable theory.  This can be contrasted with 'cosmology' which has failed and has been modified so many times, even Lemaître would not recognize it in its present form.

- Why did Halton Arp's name disappear?  He contributed amazing observational data on the association of quasars and galaxies.  After decades of work, Fulton et al. confirmed that galaxies and quasars are indeed associated [^1] with over $20\sigma$ statistical confidence, ***falsifying the redshift interpretation as an indicator of distance***.
My concern here is that ***the best argument against the Big Bang model*** has disappeared.

[^1]: C. C. Fulton, H. C. Arp, and J. G. Hartnett, “Physical association and periodicity in quasar families with SDSS and 2MRS,” Astrophysics and Space Science, vol. 363, no. 7, p. 134, Jul. 2018, doi: [10.1007/s10509-018-3355-5](https://doi.org/10.1007/s10509-018-3355-5).

**Reply by mikehelland** [reply]
>My concern with your edit is that a group called "the Thunderbolts Project" has used the excellent work done in plasma physics to promote non-scientific ideas on the nature of the electric universe. ACG should avoid at all costs any association with the "Thunderbolt Project - Electric Universe" charlatans. 'Relativity' was a better example of a stable theory.

Absolutely.

**Reply by budrap00** [reply]
@RedshiftDrift 

I agree with both of your points. QM and RT are stable theories. That does not mean they are complete or completely correct, but they are stable and useful.

The Thunderbolt Project has made plasma a dirty word in the scientific community. Even an innocuous use of the "p" word can elicit the question, "What are you an electric universe nut?" The TP made the standard neglect of plasma in the dynamics of large scale systems seem more reasonable than it is.

**Reply by sahil5d** [reply]
- Whatever ACG mentions in the opening letter, we better be able to defend fully and without exception. Many of us have identified issues with Relativity Theory (didn't you have reservations? https://github.com/orgs/a-cosmology-group/discussions/215). So I thought about modern theories, and I thought plasma physics. It's relevant as it's Lerner's expertise and as fusion companies are thriving these days. Also, plasma physics guides understanding of galactic filaments (very large scale physics), and it juxtaposes well with quantum mechanics. Alfvén's plasma research and critique of Big Bang Theory are also relevant.
- At least in my information environment, when people hear 'plasma physics' they think of the Sun, stars, lab experiments, and fusion energy.
- Re Arp, I was simply looking at the sentence and it didn't make sense to have a list like Hubble, Zwicky, Hoyle, Arp, Guth, Riess. What is the through-line there 😂? So I thought it's better to explicitly compare and write it as the era of a, b, c vs the era of x, y, z.

**Comment by ExpEarth** [comment]
I did this little logo a while back.  Meant to look a bit like a galaxy.

![image](https://github.com/a-cosmology-group/acg/assets/136624758/10f0e1eb-4dd3-4515-882a-7daaffe29d38)
